client/src/components/StreamField/blocks/StreamBlock.js

```diff
@@ -318,7 +318,12 @@ export class StreamBlock extends BaseSequenceBlock {
     // Check if there are any block types that have count limits
     this.disabledBlockTypes = new Set();
     for (const blockType in this.blockDef.meta.blockCounts) {
-      if (this.blockDef.meta.blockCounts.hasOwnProperty(blockType)) {
+      if (
+        Object.prototype.hasOwnProperty.call(
```

Comment:
Can we please replace these all with `Object.hasOwn(theObject, theThing)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn

Reply:
I see that this will satisfy the linter, but really we should be using a simpler approach here and modern browsers all support the static method `Object.hasOwn` so we do not have to dig into the prototype method.

Double check safari but pretty sure our compiler will transpile this down (we should check though)

Reply:
Yep, it should be fine as we are using `core-js` https://github.com/zloirock/core-js#ecmascript-object